Makefile

```diff
@@ -67,16 +70,28 @@ verify: fmt vet tidy generate ## verification checks against code.
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./util ./controllers -coverprofile cover.out
```

Comment:
This could cause future packages not to be tested.

Reply:
Should be fixed with commit https://github.com/operator-framework/oria-operator/commit/9cfafd2158ea14da68bb62915c3defd4878b3ba2